.github/workflows/e2e-node.yml

```
@@ -15,11 +17,19 @@ jobs:
     runs-on: ${{ matrix.os }}
     environment:
       name: ${{ matrix.environment-name }}
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
         os: [ubuntu-latest]
         node-version: [16.x]
```

our version range says we still support node.js 14.x, so we should test on that too: https://github.com/inrupt/solid-client-notifications-js/blob/main/package.json#L98
```suggestion
        node-version: [16.x, 14.x]
```